content/en/docs/ops/deployment/deployment-models/index.md

```diff
@@ -247,20 +262,85 @@ zones, or regions.
+
+## Mesh models
+
+Istio supports having all of your services in a
+{{< gloss "service mesh" >}}single mesh{{< /gloss >}}, or federating multiple meshes
+together, which is also known as {{< gloss >}}multi-mesh{{< /gloss >}}.
+
+### Single mesh
+
+The simplest Istio deployment is a single mesh. A single mesh can span
+[one or more clusters](#cluster-models) and [one or more networks](#network-models).
+All workloads in a mesh share a common certificate authority or the same trust
+domain depending on the following situations:
+
+- If you are using a vendor hosted CA that is shared across different customers,
+  then it is imperative to use a single trust domain for the entire mesh, as the
+  root CA across different customers could be the same (i.e. the vendor's CA itself).
+
+- If you have your own root CA for all your clusters, you can choose to use a single shared
+  trust domain or a separate trust domain for each cluster.
+  
+If you use a single shared trust domain, a service in a given namespace is globally unique - i.e. if
+two clusters `cluster-1` and `cluster-2` have the same service `foo` in namespace `ns`, they will be
+treated the same for purposes of security. With a trust domain for each cluster, there are no
+restrictions on the uniqueness of service and namespaces. You can apply enforcement
+policies that allow or deny all traffic from one cluster (irrespective of the namespace/service) to another.
+
+### Multiple meshes
+
+Multiple mesh deployments result from {{< gloss >}}mesh federation{{< /gloss >}}, with the following capabilities:
+
+- Organizational boundaries: lines of business.
+- Potential for cross vendor meshes to communicate with each
+  other if the trust bundles are setup properly.
+
+You can enable inter-mesh communication with {{< gloss >}}mesh federation{{</gloss >}}.
+When federating, each mesh can expose a set of services and identities, which all participating meshes can recognize.
+
+{{< image width="50%"
+    link="multi-mesh.svg"
+    alt="Multiple service meshes"
+    title="Multi-mesh"
+    caption="Multiple service meshes"
+    >}}
 
-- One cluster per region (**lowest availability**)
-- Multiple clusters per region
-- One cluster per zone
-- Multiple clusters per zone
-- Each cluster (**highest availability**)
+When federating two meshes that do not share the same
+{{< gloss >}}trust domain{{< /gloss >}}, you must
+{{< gloss "mesh federation">}}federate{{< /gloss >}}
+{{< gloss >}}identity{{< /gloss >}} and **trust bundles** between them. See the
+section on [Multiple Trust Domains](#trust-between-meshes) for an overview.
 
 ## Identity and trust models
 
 When a workload instance is created within a service mesh, Istio assigns the
 workload an {{< gloss >}}identity{{< /gloss >}}.
 
-The Certificate Authority (CA) creates and signs the certificates used to verify
+The Certificate Authority (CA) in the control plane creates and signs the certificates used to verify
```

Comment:
This wording seems to imply `citadel` (i.e. part of istiod). Was there some motivation for adding this?